.travis.yml

```diff
@@ -15,11 +15,10 @@ install:
   - pip install --upgrade tox tox-travis coveralls
 
 script:
-  - tox -e clean,codegen PRINT=1 PYTHON_CMD=python
-  - tox
+  - tox -e clean,build_test,codegen PRINT=1 PYTHON_CMD=python
+  - tox -e test
```

Comment:
By explicitly setting -e test, this skips flake8 and docs on py36

Reply:
That's not good.
Fixed